source/Cosmos.VS.ProjectSystem/BuildSystem/Rules/CosmosPropertyPage.xaml

```diff
@@ -62,6 +62,12 @@
                   Category="Compile">
     </BoolProperty>
 
+    <BoolProperty Name="RemoveBootDebugOutput"
+                  DisplayName="Enable Little Optimization"
```

Comment:
Can you please choose a more informative display name and description?

Reply:
Thanks, I forgat about it.
Now i fixed it